content/lessons/chapter-3/coop-1.tsx

```diff
@@ -5,65 +5,39 @@ import { Title } from 'ui'
 import { Button } from 'shared'
 import Avatar from 'components/Avatar'
 import { useMediaQuery } from 'hooks'
-import { useState, useEffect } from 'react'
+import { useEffect } from 'react'
+import ProfileChip, { ProfileChipVariant } from 'components/ProfileChip'
+import Image from 'next/image'
 import { cssVarThemeChange } from 'lib/themeSelector'
 
 export const metadata = {
   title: 'chapter_three.coop_one.title',
-  image: '/assets/images/chapter-2-intro-1.jpg',
   key: 'CH3COO1',
 }
 
-export default function Coop1({ lang }) {
+export default function Pool1({ lang }) {
```

Comment:
This is a bit confusing, the component is named Pool1 but the file is named coop-1. If this component is meant to be for pool-1 can we not switch this code with whatever is in pool-1.tsx and update the routing accordingly?